experiments/testing_archive/experiment_tests.csv

```diff
@@ -0,0 +1,155 @@
+# The following tests don't seem to be testable by writing an experiment file. Some of them are
+# testable by other methods.
+#
+# Class ExperimentPrototype:
+#
+# method set_scheduling_mode:
+#    1. scheduling_mode set by experiment handler is not in the possible_scheduling_modes list
+# method del_slice:
```

Comment:
although it might not be a common thing to do, I think you could add_slice and then del_slice all within an experiment init? Similar with edit_slice